problems/removeOddNumbers.js

```diff
@@ -2,8 +2,14 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - The input array with all odd number removed
 */
-function removeOddNumbers() {
-
+function removeOddNumbers(arr) {
+    for (let i = 0; i < arr.length; i++) {
+        while (arr[i] % 2) {
```

Comment:
This is a little wild. Typically we want to avoid double loops / mutating an array while iterating through it. Pleas refactor with filter.